docs/site/Using-components.md

```diff
@@ -31,24 +27,3 @@ const app = new RestApplication();
 // authenticated requests in a Sequence.
 app.component(AuthenticationComponent);
 ```
-
-Components can contribute the following items:
-
-- [Controllers](Controllers.md)
-- Providers of additional [Context values](Context.md)
-
-LoopBack 4 was built with extensibility in mind and this includes Components,
-which can be allowed to contribute additional artifacts by adding a Mixin to
-your Application class. This doesn't change how a Component is registered
-(`app.component()`) but it enables the Component to contribute additional
-artifacts. For example:
-
-- [Repositories](Repositories.md) can be contributed by a Component by adding
-  `RepositoryMixin` from `@loopback/repository` to your Application
-- [Booters](Booting-an-Application.md#booters) can be contributed by a Component
-  by adding `BootMixin` from `@loopback/boot` to your Application
-
-**Note:** Always check a component's instructions to see if it requires the use
-of a Mixin. A Mixin may automatically register a Component, saving you the
-trouble of having to do so manually. Again it's best to check the documentation
-for the given Component / Mixin.
```

Comment:
+1 to simplify this page. I am afraid you went too far here. When people open "Using Components", there is no (easy) way how to find out more details about what a component is and what it can contribute. Please add a short sentence with a link to the new `Components` page.

For example, at the top:

```markdown
Components serve as a vehicle to group 3rd-party contributions to allow easier
extensibility of your Application, see [Components](./Components.md) for more details.

Components can be added to your application using the `app.component()` method.
```